webui/worker/src/constants/data/routePaths.tsx

```diff
@@ -15,5 +15,6 @@ export const routePaths = {
   blockInfo: '/blockInfo',
   logs: '/logs',
   metrics: '/metrics',
+  stacks: '/stacks',
```

Comment:
Do you mean to `"/api/v1/common/thread_dump`?

Reply:
These two path values should be different.